components/raftstore/src/store/msg.rs

```diff
@@ -811,7 +811,23 @@ impl<EK: KvEngine> ResourceMetered for PeerMsg<EK> {}
 impl<EK: KvEngine> fmt::Debug for PeerMsg<EK> {
     fn fmt(&self, fmt: &mut fmt::Formatter<'_>) -> fmt::Result {
         match self {
-            PeerMsg::RaftMessage(..) => write!(fmt, "Raft Message"),
+            PeerMsg::RaftMessage(m, _) => {
```

Comment:
What about impl Debug for InspectedRaftMessage to avoid repeating this code block.